test/flows/check_uk_visa_flow_test.rb

```diff
@@ -639,16 +639,16 @@ class CheckUkVisaFlowTest < ActiveSupport::TestCase
       assert_rendered_outcome text: "You may want to apply for a transit visa"
     end
 
-    should "not render a suggestion of applying for a transit visa for non-ETA countries" do
+    should "render a suggestion of applying for a transit visa for EU/EEA ETA countries" do
       add_responses what_passport_do_you_have?: @eea_country,
                     travelling_to_cta?: "somewhere_else"
-      assert_no_rendered_outcome text: "You may want to apply for a transit visa"
+      assert_rendered_outcome text: "You may want to apply for a transit visa"
     end
 
     should "render a suggestion of a visa for a further journey to Ireland" do
```

Comment:
Should the test name change to "transit visa", to match the test above?
```suggestion
    should "render a suggestion of a transit visa for a further journey to Ireland" do
```